CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.8.0
+
+v1.8.0 is a maintenance release with the following fixes and enhancements:
+
+ - **Python 2.7 binary wheels are no longer provided.**
+   Users still on Python 2.7 will need to build confluent-kafka from source
+   and install librdkafka separately, see [README.md](README.md#Prerequisites)
+   for build instructions.
+ - Added `use.latest.version` and `skip.known.types` (Protobuf) to
```

Comment:
i don't think the protobuf framing incompatibility got fixed?

Reply:
Is there a PR for it that Ive missed?